week01/leetcode274.cpp

```diff
@@ -0,0 +1,15 @@
+// 274. H-Index >> https://leetcode.com/problems/h-index/
+// Time: O(NlogN) where N is the size of the array
+// Space: O(1) 
+
+class Solution {
+public:
+    int hIndex(vector<int>& citations) {
+        sort(citations.begin(), citations.end(), greater<int>());
+        int i;
+        for(i = 0; i < citations.size(); i++) {
```

Comment:
```suggestion
        for(int i = 0; i < citations.size(); i++) {
```

C++ supports variable declaration in for loops.